### DISCUSSION TITLE
How to try out Tailwind 4 Beta?

### DISCUSSION THREAD

**Discussion by gobijan** [rails/tailwindcss-rails#439] [General]
What is the best way to switch to Tailwind 4 Beta? 
Tailwindcss Ruby already has a release for the latest Betas. 
https://github.com/flavorjones/tailwindcss-ruby/releases/tag/v4.0.0.beta.2

So just use TAILWINDCSS_INSTALL_DIR env variable for a locally installed V4 Beta?

Thx

**Comment by flavorjones** [comment]
@gobijan Thanks for asking. To try out the prerelease, just specify it in your Gemfile like:

```
gem "tailwindcss-ruby", "4.0.0.beta.2"
```

**Reply by flavorjones** [reply]
Though, please note that it's not going to Just Work™. Please make sure you read [Tailwind CSS v4.0 Beta - Tailwind CSS](https://tailwindcss.com/docs/v4-beta) and let us know how you get on!

**Reply by gobijan** [reply]
Can confirm that it works for me using "4.0.0.beta.3".

Slimlined my tailwind.config.js:

const defaultTheme = require('tailwindcss/defaultTheme')

    module.exports = {
      content: [
        './public/*.html',
        './app/helpers/**/*.rb',
        './app/javascript/**/*.js',
        './app/views/**/*.{erb,haml,html,slim}'
      ],
    }

Didn't test if their new default heuristics catch erb files but they for sure not catch .gohtml

**Comment by flavorjones** [comment]
Related, please see https://github.com/rails/tailwindcss-rails/discussions/450 for Tailwind 4 upgrade experience reports.
